### DISCUSSION TITLE
Tree-shaking PrismJS and HighlightJS dependencies

### DISCUSSION THREAD

**Discussion by hoanvh** [webdevnerdstuff/vue-code-block#57] [Ideas]
Thanks for your great component.

In my application, when applying VCodeBlock, I'm only using PrismJS library, without any HighlightJS features.
However, when building a bundle, HighlightJS dependency was built also, and to be honest, it's so heavy (~300kb after gzip).

It would be great if we will pack only library we will use

**Comment by webdevnerdstuff** [comment]
I kind of wish I stuck with my original plan on having two versions of the plugin, one for PrismJs and the other for Highlight.js so I could separate things a little. I just didn't want to have to maintain two separate packages.

I'm not sure why Highlight.js is bundling for you, as it's code is not in my compiled code as it's considered [excluded](https://github.com/webdevnerdstuff/vue-code-block/blob/main/vite.build.config.mts#L41). Do you have both Prism and Highlight.js installed in your package.json?
